routes/web.php

```diff
@@ -21,10 +21,8 @@
 |
 */
 
-Route::get('/', function () {
-    // welcome and sign-up page.
-    return Inertia::render('Home');
-})->name('/');
+// if you go to tasklab.site.com/ i think it should be the registration page
+Route::get('/',  [RegisterController::class, 'showRegister'])->name('/');
```

Comment:
Idk about this. Then for our landing page it would have to have a path other than '/'. If you look at Proggio, monday, and asana they reserve the '/' path for the landing page. One site opens a modal for signups, but the others use /sign-up

Reply:
Sounds good. After looking around a little bit more I see what you mean